core/vm/contracts.go

```diff
@@ -93,7 +93,8 @@ func ecrecoverFunc(in []byte) []byte {
 	vbig := common.Bytes2Big(in[32:64])
 	v := byte(vbig.Uint64())
 
-	if !crypto.ValidateSignatureValues(v, r, s) {
+	// tighter sig s values in homestead only apply to tx sigs
```

Comment:
This is wrong actually. I'm modifying the code so that this isn't ignored in the EVM. I'm not sure why this was done in the first place.